modules/src/ics03_connection/msgs/conn_open_init.rs

```diff
@@ -22,6 +22,7 @@ pub struct MsgConnectionOpenInit {
     pub connection_id: ConnectionId,
     pub client_id: ClientId,
     pub counterparty: Counterparty,
+    pub version: String,
```

Comment:
It would be nice to introduce a newtype for versions across the codebase, but that can be done as part of #306.